src/navigation/tabs/__test__/tabsMockUps.js

```diff
@@ -2,7 +2,7 @@ export const items = [1, 2, 3, 4, 5, 6, 7, 8].map((tabNumber) => ({
     getContent: () => `Example Tab ${tabNumber} content`,
     key: `exampleTab${tabNumber}`,
     // eslint-disable-next-line no-console
-    onClick: (clickedTab) => { console.log(`Example Tab ${tabNumber} clicked`, clickedTab); },
+    onClick: (clickedTab) => console.log(`Example Tab ${tabNumber} clicked`, clickedTab),
```

Comment:
Is the `console.log()` useful for the test?  If not, let's remove it and maybe go with `jest.fn()` or `noop` or something.  If so, so we need to suppress ESLINT issue?

Reply:
Used `noop`

Reply:
Beauty.  Resolving.